src/certificate.c

```diff
@@ -93,6 +106,8 @@ static PyMethodDef x509_methods[] = {
 	 "get a dict of oid: value"},
 	{"as_pem", (PyCFunction) as_pem, METH_VARARGS,
 	 "return the pem representation of this certificate"},
+	{"as_text", (PyCFunction) as_text, METH_VARARGS,
+	 "return the text representation of this certificate"},
```

Comment:
I think I would clarify this a little since the only universal "text" representation of a certificate is the PEM.  I'm assuming this method is referring more to what `openssl x509 -in blah.pem -noout -text` returns?